src/Symfony/Component/Notifier/Bridge/Twitter/TwitterTransport.php

```diff
@@ -0,0 +1,297 @@
+                'total_bytes' => $file->getSize(),
+                'media_type' => $file->getContentType(),
+            ];
+            if ($category) {
+                $query['media_category'] = $category;
+            }
+            if ($extraOwners) {
+                $query['additional_owners'] = implode(',', $extraOwners);
+            }
+
+            $pool[++$i] = $this->request('POST', '/1.1/media/upload.json', [
+                'query' => $query,
+                'user_data' => [$i, null, 0, fopen($file->getPath(), 'r'), $alt, $subtitles],
+            ]);
+
+            if ($subtitles) {
+                $query['total_bytes'] = $subtitles->getSize();
+                $query['media_type'] = $subtitles->getContentType();
+                $query['media_category'] = 'subtitles';
+
+                $pool[++$i] = $this->request('POST', '/1.1/media/upload.json', [
+                    'query' => $query,
+                    'user_data' => [$i, null, 0, fopen($subtitles->getPath(), 'r'), null, $subtitles],
+                ]);
+            }
+        }
+
+        $mediaIds = [];
+        $subtitlesVideoIds = [];
+        $subtitlesMediaIds = [];
+
+        try {
+            while ($pool) {
+                foreach ($this->client->stream($pool) as $response => $chunk) {
+                    $this->processChunk($pool, $response, $chunk, $mediaIds, $subtitlesVideoIds, $subtitlesMediaIds);
+                }
+            }
+        } catch (\Throwable $e) {
+            foreach ($pool as $response) {
+                $response->cancel();
+            }
+
+            throw $e;
+        }
+
+        foreach (array_filter($subtitlesVideoIds) as $videoId => $subtitles) {
+            $name = pathinfo($subtitles->getFilename(), \PATHINFO_FILENAME);
+            $subtitlesVideoIds[$videoId] = $this->request('POST', '/1.1/media/subtitles/create.json', [
+                'json' => [
+                    'media_id' => $videoId,
+                    'media_category' => 'tweet_video',
+                    'subtitle_info' => [
+                        'subtitles' => [
+                            [
+                                'media_id' => array_search($subtitles, $subtitlesMediaIds, true),
+                                'language_code' => pathinfo($name, \PATHINFO_EXTENSION),
+                                'display_name' => pathinfo($name, \PATHINFO_FILENAME),
+                            ],
+                        ],
+                    ],
+                ],
+            ]);
+        }
+
+        return $mediaIds;
+    }
+
+    private function processChunk(array &$pool, ResponseInterface $response, ChunkInterface $chunk, array &$mediaIds, array &$subtitlesVideoIds, array &$subtitlesMediaIds): void
+    {
+        if ($chunk->isFirst()) {
+            $response->getStatusCode(); // skip non-2xx status codes
+        }
+        if (!$chunk->isLast()) {
+            return;
+        }
+
+        if (400 <= $response->getStatusCode()) {
```

Comment:
```suggestion
        if ($chunk->isFirst()) {
            $response->getStatusCode(); // skip non-2xx status codes
        }

        if (!$chunk->isLast()) {
            return;
        }

        if (400 <= $response->getStatusCode()) {
```

Missing empty line - imho - We should use same style with these.